GetOffMyLawn/GetOffMyLawn.cs

```diff
@@ -131,6 +138,14 @@ class PiecePatch {
 
         __instance.m_nview.GetZDO().Set(_healthHashCode, _pieceHealth.Value);
       }
+
+      [HarmonyPostfix]
+      [HarmonyPatch(nameof(Piece.Awake))]
+      static void CanBePremovedPostfix(ref Piece __instance) {
+        if (_removablePieceOverrides.Contains(__instance.m_name)) {
+          __instance.m_canBeRemoved = true;
+        }
+      }
     }
```

Comment:
Wasn't sure of the best way to tap into this

![image](https://user-images.githubusercontent.com/656630/153332238-36cb3c01-c21b-403c-b621-eb212a43dc79.png)

Seems like an opportunity for transpiling, but that's out of my ballgame

Reply:
Actually i see where in the IL instructions I'd tap into it... think this is the way?